pkg/csi-clients/identity/identity_fake.go

```diff
@@ -62,7 +62,7 @@ func (m *mockIdentity) GetMigrationCapabilities(_ context.Context) (MigrationCap
 
 func getSupportedActions() (ReplicationCapabilitySet, []*replication.SupportedActions) {
 	capResponse := &replication.GetReplicationCapabilityResponse{}
-	for i := 0; i < 3; i++ {
+	for i := int32(0); i < 3; i++ {
```

Comment:
Was this flagged by the new linter? Seems to break a popular idiom.

Reply:
Yes, it was complained recently.